Makefile

```
@@ -112,25 +116,25 @@ manifests.single: ## Compose single-file deployment manifests from building bloc
 # ------------------------------------------------------------------------------
 
 .PHONY: generate
-generate: generate.controllers generate.clientsets controller-gen 
-	$(CONTROLLER_GEN) object:headerFile="hack/boilerplate.go.txt" paths="./..."
+generate: generate.controllers generate.clientsets
 
 .PHONY: generate.controllers
-generate.controllers:
+generate.controllers: controller-gen
 	go generate ./...
+	$(CONTROLLER_GEN) object:headerFile="hack/boilerplate.go.txt" paths="./..."
```

These should actually be flipped:

```suggestion
	$(CONTROLLER_GEN) object:headerFile="hack/boilerplate.go.txt" paths="./..."
	go generate ./...
```